packages/flutter_driver/BUILD.gn

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 The Flutter Authors. All rights reserved.
```

Comment:
I think we still need to use "The Chromium Authors" (/cc @sethladd dyk?).

If that's the case, can you send a separate PR to update the license headers in `fuchsia_remote_debug_protocol`?

Reply:
Double-checked and it looks to be the case from a previous code review.  I'll update frdb